src/sentry/uptime/models.py

```diff
@@ -0,0 +1,65 @@
+from datetime import timedelta
+from typing import ClassVar, Self
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BaseManager,
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class ProjectUptimeSubscription(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    project_id = BoundedBigIntegerField(db_index=True)
+    uptime_subscription = FlexibleForeignKey("sentry.UptimeSubscription")
+
+    objects: ClassVar[BaseManager[Self]] = BaseManager(
+        cache_fields=["pk"], cache_ttl=int(timedelta(hours=1).total_seconds())
+    )
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_projectuptimesubscription"
+
+        constraints = [
+            models.UniqueConstraint(
+                fields=["project_id", "uptime_subscription"],
+                name="sentry_projectuptimesubscription_unique_project_subscription",
+            ),
+        ]
+
+
+@region_silo_model
+class UptimeSubscription(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    remote_subscription_id = BoundedBigIntegerField(unique=True)
```

Comment:
Why can't we use flexible foreign key here?

Reply:
Because I think we'll put `RemoteSubscription` in the primary and `UptimeSubscription` in crons

Reply:
Does it matter? There aren't going to be a lot of uptimeSubscriptions right? Or are you anticipating another big check-in table?

Reply:
I just assumed we'd put all our stuff in the database our team owns, but I don't feel too strongly. When we have a checkin table it'll be in clickhouse so that's not a concern for me